docs/manual/running.md

```diff
@@ -2,10 +2,100 @@
 
 # Running simulations
 
-:::{todo}
+The :program:`remage` executable drives the simulation from the command line. A
+short overview of all available options can be obtained with `remage --help`.
 
-- describe command line options
-- batch versus interactive mode
-- multithreading
+```console
+$ remage [OPTIONS] [macros or command string...]
+```
+
+## Command line options
+
+Typical invocations supply one or more GDML geometry files and a list of macros
+to execute:
+
+```console
+$ remage -g setup.gdml -- run.mac
+```
+
+:::{note}
+
+The `--` is required here to disambiguate since `-g` can be supplied with
+multiple arguments.
 
 :::
+
+The most useful options include:
+
+- `macros` – macro files or inline command strings. If the given argument is not
+  an existing file it is interpreted as a newline, semicolon, or comma separated
+  list of commands.
+- `-g, --gdml-files` – include one or more GDML geometry files (see
+  {ref}`geometry`).
+- `-o, --output-file` – sensitive detector hits output file (see {ref}`output`).
+- `-i, --interactive` – keep the application open after executing macros and
+  present a Geant4 prompt.
+- `-t, --threads` – number of worker threads to use.
+- `-w, --overwrite` – overwrite an existing output file.
+- `-q, --quiet`/`-v, --verbose`/`-l, --log-level` – control the verbosity.
+  Logging levels are `debug`, `detail`, `summary`, `warning`, `error`, `fatal`,
+  and `nothing` (e.g. `-l debug`).
+- `--version`/`--version-rich` – print version information and exit.
+- `-m, --merge-output-files` – merge thread-specific output files at the end of
+  execution (see {ref}`output`).
+- `--flat-output` – store each Geant4 step as its own row in the output (see
+  {ref}`output`).
+- `--time-window-in-us` – time window used when reshaping hits (see
+  {ref}`output`).
+- `-s, --macro-substitutions` – provide `key=value` pairs that will be expanded
+  as Geant4 aliases in macros.
+
+## Batch versus interactive mode
+
+By default :program:`remage` runs all specified macro commands in batch mode and
```

Comment:
program links don't work like that (they reference a RST-defined CLI documentation, which we don't have). Also this is not the correct myst syntax.

Reply:
This is the AI making up stuff. I'll fix